knowledge-graph/src/main/scala/ch/datascience/knowledgegraph/datasets/rest/BaseDetailsFinder.scala

```diff
@@ -58,7 +62,7 @@ private class BaseDetailsFinder(
       renku  -> "renku",
       schema -> "schema"
     ),
-    s"""|SELECT DISTINCT ?identifier ?name ?maybeDateCreated ?alternateName ?url ?topmostSameAs ?maybeDerivedFrom ?initialVersion ?description ?maybePublishedDate
+    s"""|SELECT DISTINCT ?identifier ?name ?maybeDateCreated ?alternateName ?url ?topmostSameAs ?maybeDerivedFrom ?initialVersion ?description ?maybePublishedDate ?projectId ?projectName ?minDateCreated ?maybeAgentEmail ?agentName
```

Comment:
Something here isn't working. I'm getting: 

```
TestFailedException was thrown during property evaluation.
  Message: Left(DecodingFailure(Attempt to decode value on failed cursor, List(DownField(projectId), DownArray, DownField(bindings), DownField(results)))) was not equal to Right(List(NonModifiedDataset(29.8296/zenodo.4068900,tetlozWlr,fwu,https://Hcmjju:5571/uscurv/avcer,http://gdw:2069/_-slxh_lp/ipyvBC_jhF/datasets/29.8296%2Fzenodo.4068900,DatasetVersions(29.8296/zenodo.4068900),None,Set(),RenkuDatasetDates(2016-07-16T12:09:46.805Z),List(),DatasetProject(ccb2mb/3cdofh/hk18_-5Rp/cDkneoVh/ytt0x.6uo_d,wcUVyjxgu,AddedToProject(1973-09-14T03:17:10.011Z,DatasetAgent(Some(ba*vn2w*45@amis),ziMxkfq gdlzih))),List(),List(),List())))
  Location: (BaseDetailsFinderSpec.scala:48)
  Occurred when passed generated values (
    arg0 = NonModifiedDataset(29.8296/zenodo.4068900,tetlozWlr,fwu,https://Hcmjju:5571/uscurv/avcer,http://Blngiuueq:2880/ZmqN,DatasetVersions(29.8296/zenodo.4068900),Some(irwjm Kt clbraby dwlc aty crxPds As yd LuoweyG iinsq),Set(DatasetCreator(Some(s9oE5xa@honln),Xzk` /`,Some(rqDxtrKga)), DatasetCreator(Some(t~3j6O=i@ovqvxq),qwobK Knxeaq,Some(mmy)), DatasetCreator(Some(K+&bhq@cfHdh),h`kczY-rbq nzbr,Some(uMlkyqre))),RenkuDatasetDates(2016-07-16T12:09:46.805Z),List(DatasetPart(wtIojwgwd,data/iv-d/r-nvpnn_g), DatasetPart(vxbaqmcbl,data/qhiwccbs/jgdkl-v)),DatasetProject(ccb2mb/3cdofh/hk18_-5Rp/cDkneoVh/ytt0x.6uo_d,wcUVyjxgu,AddedToProject(1973-09-14T03:17:10.011Z,DatasetAgent(Some(ba*vn2w*45@amis),ziMxkfq gdlzih))),List(DatasetProject(6vc0xl-s/zuinx0.6/bY.Am-M65r/mms9qijd5,lkovuczbwq,AddedToProject(2001-12-09T20:22:52.491Z,DatasetAgent(Some(oq25m6k@itKt),uurf xqMprk))), DatasetProject(ehpj_86cv/wvwbych_c-j/4dchuo,aedceec,AddedToProject(2003-09-18T00:39:01.317Z,DatasetAgent(Some(kotqVpklNXr@uaFam),r-ma/um-zn jeekoge-r))), DatasetProject(_2fmhngmrEl/d6kxd9eal_p,xvjxsvee,AddedToProject(1979-07-19T10:21:11.171Z,DatasetAgent(Some($p~$ac98hu+@aeqsbu),ndzq/Glyr p\/))), DatasetProject(okezt_/uzqycdi_/gae_v253I3/g7ohewbd/y6eduvj8nha,obvpgdqt,AddedToProject(1979-11-15T23:39:55.041Z,DatasetAgent(Some(g9iyu0@YrzfNCnj),jbptf wz/ikctd))), DatasetProject(yfg0_u_3a/f.juc_/7djon3ayrfF/vx09m_s/ev6.5eovZy,TMcKwd,AddedToProject(1996-06-05T13:56:41.966Z,DatasetAgent(Some(ggtj&iw6/y@kgzhIIh),\ca_ Hswckn)))),List(peptmu, sgbanewnb, Xujwv),List()),
    arg1 = "" // 4 shrinks
  )
```

Reply:
I think the query could be missing variables needed to construct the project. I mean you select these ?projectId ?projectName ?minDateCreated ?maybeAgentEmail ?agentName but as far I can see you do not define them in the WHERE clause. I'll give you a hand tomorrow.